tests/fork_func/check_exit.c

```diff
@@ -0,0 +1,45 @@
+#include <sys/types.h>
+
+#include "fork_func.h"
+
+#include "check.h"
+
+static int
+func_exit(void * cookie)
+{
+	int exitcode = *((int *)cookie);
```

Comment:
I would write this as

```
int *exitcodep = (int *)cookie;

return (*exitcodep);
```

Does the same thing, but we generally follow a convention of "pointer = cookie" at the start of callback functions.

Reply:
fixed.